Documentation/contributing/development/debugging.rst

```diff
@@ -485,7 +485,7 @@ Inspecting profiles can help identify CPU bottlenecks and memory leaks.
 
 To capture a profile, take a :ref:`sysdump <sysdump>` of the cluster with the
 Cilium CLI or more directly, use the ``cilium-bugtool`` command that is
-included in the Cilium image:
+included in the Cilium image after enabling ``pprof`` in the Cilium ConfigMap:
```

Comment:
huh - surprising that we need this; doesn't `kubectl exec <cilium-pod> -- gops pprof-cpu 1` work out of the box?

Reply:
That's a different way to collect pprof traces. The bugtool command uses connects to the pprof server port (and not gops) by default.